universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -433,6 +434,21 @@ aws.s3.bucket=civiform-local-s3
 aws.s3.bucket=${?AWS_S3_BUCKET_NAME}
 aws.local.endpoint="http://localstack:4566"
 
+## Azure
+azure.blob.account=devstoreaccount1
+azure.blob.account=${?AZURE_STORAGE_ACCOUNT_NAME}
+azure.blob.container=civiform-local-blob
+azure.blob.container=${?AZURE_STORAGE_ACCOUNT_CONTAINER}
+azure.local.connection="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;"
+azure.local.connection=${?AZURE_LOCAL_CONNECTION_STRING}
+
+## Java
+java.time.zoneid="America/Los_Angeles"
```

Comment:
question (blocking): what is this needed for?

Reply:
ZoneID is needed for generating the SAS. SAS takes in an OffSetDateTime that's the expiration date/time for the SAS. In order to get the currnet time, we have to specify the time zone we're in.

Reply:
Ah, will this be an issue when the app is hosted and accessed on the east coast?

Reply:
it shouldn't be, as long as we set this config option to whatever timezone the application is hosted in.

Reply:
Great. Let's make this a configurable setting then. Different instances will be deployed in different time zones.